cli/command/image/build.go

```diff
@@ -543,11 +544,11 @@ func rewriteDockerfileFrom(ctx context.Context, dockerfile io.Reader, translator
 	return buf.Bytes(), resolvedTags, scanner.Err()
 }
 
-// replaceDockerfileTarWrapper wraps the given input tar archive stream and
+// replaceDockerfileForContentTrust wraps the given input tar archive stream and
 // replaces the entry with the given Dockerfile name with the contents of the
 // new Dockerfile. Returns a new tar archive stream with the replaced
```

Comment:
Worth to add content trust in the function description?

Reply:
Updated the godoc